src/models/Deck.test.ts

```diff
@@ -26,63 +26,26 @@ describe("#getCards", () => {
   });
 });
 
-describe("#drawAfterTurn", () => {
```

Comment:
🖊️  &nbsp; Moving away from `drawSingle` and `drawAfterTurn` to just `draw` with a maximum